deploy/gcloud/gcloud_up.sh

```diff
@@ -0,0 +1,186 @@
+fi
+
+if [ -f "amppackager_cert_consumer.yaml" ]; then
+  echo "amppackager_cert_consumer.toml exists. Skipping generation."
+else
+  echo "Generating consumer YAML file ..."
+  cat amppackager_cert_consumer_template.yaml | sed 's/$(PROJECT_ID)/'$PROJECT_ID'/g' | sed 's/$(AMP_PACKAGER_VERSION_TAG)/'$AMP_PACKAGER_VERSION_TAG'/g' > amppackager_cert_consumer.yaml
+fi
+
+# Build docker images for the Amppackager renewer and consumer, if necessary.
+# Renewer and consumer are the same binaries, passed different command line
+# arguments. If you don't want to rebuild the binaries, you can comment out the
+# next 2 lines as well as the docker images and push commands below.
+docker build -f Dockerfile.consumer -t gcr.io/${PROJECT_ID}/amppackager:${AMP_PACKAGER_VERSION_TAG} .
+docker build -f Dockerfile.renewer -t gcr.io/${PROJECT_ID}/amppackager_renewer:${AMP_PACKAGER_VERSION_TAG} .
+
+# To check that it succeeded, list the images that got built.
+docker images
+
+# https://cloud.google.com/container-registry/docs/advanced-authentication#gcloud-helper
+gcloud auth configure-docker
+
+# Push the docker image into the cloud container registry.
+# See: https://cloud.google.com/container-registry/docs/overview
+# See: https://cloud.google.com/container-registry/docs/pushing-and-pulling
+docker push gcr.io/${PROJECT_ID}/amppackager:${AMP_PACKAGER_VERSION_TAG}
+docker push gcr.io/${PROJECT_ID}/amppackager_renewer:${AMP_PACKAGER_VERSION_TAG}
+
+gcloud config set project $PROJECT_ID
+gcloud config set compute/zone $COMPUTE_ENGINE_ZONE
+
+# Allow 10 nodes maximum for this cluster.
+#gcloud container clusters create amppackager-cluster --tags allow-8080 --num-nodes=10
+echo "Creating kubernetes cluster"
+gcloud container clusters create amppackager-cluster --num-nodes=10 --enable-ip-alias --metadata disable-legacy-endpoints=true
+
+# Setup your credentials
+# https://cloud.google.com/sdk/gcloud/reference/container/clusters/get-credentials
+gcloud container clusters get-credentials amppackager-cluster
+
+# Create the NFS disk for RW sharing amongst the kubernetes deplouments
+# cert-renewer and cert-consumer.
+# https://medium.com/@Sushil_Kumar/readwritemany-persistent-volumes-in-google-kubernetes-engine-a0b93e203180
+echo "Creating NFS disk"
+gcloud compute disks create --size=10GB --zone=${COMPUTE_ENGINE_ZONE} amppackager-nfs-disk
+kubectl apply -f nfs_renewer_pvc.yaml
+kubectl apply -f nfs_consumer_pvc.yaml
+kubectl apply -f nfs_clusterip_service.yaml
+kubectl apply -f nfs_server_deployment.yaml
+export AMPPACKAGER_NFS_SERVER=$(kubectl get pods | grep amppackager-nfs | awk '{print $1}')
+
+# Sleep for a few minutes, waiting for NFS disk to be deployed.
+sleep 4m
+
+# This assumes current working directory is amppackager/docker/gcloud
+# default is the default namespace for the gcloud project
+echo "Copying files to NFS mount ..."
+kubectl cp www default/"$AMPPACKAGER_NFS_SERVER":/exports/
+
+if [ -f "amppkg_consumer.toml" ]; then
```

Comment:
Why put the toml files on NFS? Can't they go in the docker container directly?

Reply:
I wanted to give the users the flexibility to modify the TOML files in the NFS mount.  It'll be harder to do this if the files are baked into the docker image.